core/main/src/state/metrics_state.rs

```diff
@@ -342,7 +357,14 @@ impl MetricsState {
                 .await
             {
                 Some(s) => s,
-                None => state.get_device_manifest().get_form_factor(),
+                None => {
+                    let form_factor = state.get_device_manifest().get_form_factor();
```

Comment:
Oops, I think I gave you bad advice on doing the mapping here, because we can't have any SIFT-related stuff in the `Ripple` repo... I think those references need to go back in `ripple-eos-distributor-extn` where you had them, maybe do the mapping when the `OntologyPayload` is assembled.